js/bitmex.js

```diff
@@ -1821,7 +1910,7 @@ module.exports = class bitmex extends Exchange {
         const request = {
             'symbol': market['id'],
             'side': this.capitalize (side),
-            'orderQty': parseFloat (this.amountToPrecision (symbol, amount)), // lot size multiplied by the number of contracts
+            'orderQty': this.quantityConversion (market, amount),
```

Comment:
do you think it we could add an `legacyPrecision`option that users could enable to avoid a breaking change?

Reply:
oh, i think I had restructured whole file, there was no single point of precision corrections, instead whole exchange/methods  were amended. 
tomorrow I will take a look into this.

Reply:
Carlos, I've updated few things to further fix precisions and parse them correctly through input/output.
However, as I remind, the pre-PR version of bitmex was so different, as if you check the current changes vs initial, you will see that almost whole exchange and blocks of codes inside each method, have been reconstructed, so retaining the previous version would be need re-adding the pre-PR codes (which will increase the code massively and might make it sphagetti-like).
I understand that breaking change is not desirable, but having both versions are not easy in bitmex.
please lmk your thoughts.